src/PyCall.jl

```diff
@@ -663,11 +664,12 @@ function pybuiltin(name)
 end
 
 #########################################################################
+include("pycalls.jl")
 
 """
 Low-level version of `pycall(o, ...)` that always returns `PyObject`.
 """
-function _pycall(o::Union{PyObject,PyPtr}, args...; kwargs...)
+function _pycall_legacy(o::Union{PyObject,PyPtr}, args...; kwargs...)
```

Comment:
This is just here for the benchmarks, the whole function, and the `pycall_legacy` methods below, need to be removed before merging.